makefile

```
@@ -1,49 +1,47 @@
+
+# The final build step.
+$(BUILD)/$(TARGET): $(OBJS)
+	@$(CC) -std=$(STD) $(CFLAGS) $(DFLAGS) $(OBJS) -o $@ $(LDFLAGS)
+
+# Build step for C source
+$(BUILD)/%.c.o: %.c
+	@echo "Compiling: '$<'"
+	@mkdir -p $(dir $@)
+	@$(CC) -std=$(STD) $(CPPFLAGS) $(DFLAGS) $(CFLAGS) -c $< -o $@
+
+.PHONY: clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)